tests/common/dualtor/control_plane_utils.py

```diff
@@ -57,20 +57,22 @@ def expect_app_db_values(duthost, intf_name, state):
     db = APP_DB
     mux_states = {}
     match = True
-    for table, field in APP_DB_MUX_STATE_FIELDS.items():
-        key = table + "|" + intf_name
-        _keys(duthost, db, key)
-        mux_states[key] = _hgetall(duthost, db, key)
-        if mux_states[key][field] != state:
-            match = False
+    for intf_name in intf_names:
+        mux_states[intf_name] = {}
+        for table, field in APP_DB_MUX_STATE_FIELDS.items():
+            key = table + "|" + intf_name
+            _keys(duthost, db, key)
+            mux_states[intf_name][key] = _hgetall(duthost, db, key)
```

Comment:
do we need to have the 'key' in the dict? 
mux_states[intf_name] should be good enough